app/controllers/evaluation_forms_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 # Controller for evaluation forms CRUD actions.
 class EvaluationFormsController < ApplicationController
+  helper FormHelper
```

Comment:
this isn't necessary, the helpers are automatically included on every controller